RotationMatrix rotation

#/** translation as a Vector3*/
Vector3 translation
